app/airq/commands/feedback.py

```diff
@@ -0,0 +1,26 @@
+import typing
+
+from airq import config
+from airq.commands.base import ApiCommandHandler
+from airq.lib.ses import send_email
+from airq.models.clients import Client
+from airq.models.events import EventType
+
+
+class ShowFeedbackHandler(ApiCommandHandler):
+    def handle(self) -> typing.List[str]:
+        message = ["Please enter your feedback below:"]  # consider adding cancel option
+        self.client.log_event(EventType.FEEDBACK_BEGIN)
+        return message
+
+
+class ReceiveFeedbackHandler(ApiCommandHandler):
+    @classmethod
+    def should_handle(cls, pattern: str, client: Client, user_input: str) -> bool:
+        return client.should_accept_feedback()
+
+    def handle(self) -> typing.List[str]:
+        send_email(config.ADMIN_EMAILS, "User gave feedback", self.user_input)
+        message = ["Thank you for your feedback!"]
```

Comment:
You could refactor this to be a bit neater:

```python3
send_email(config.ADMIN_EMAILS, "User gave feedback", self.user_input)
self.client.log_event(EventType.FEEDBACK_RECEIVED, feedback=self.user_input)
return ["Thank you for your feedback!"]
```

Reply:
All fo the other commandHandlers memorialize message into a variable even when they don't modify it, so I stuck to that convention